<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agregar</class>
 <widget class="QDialog" name="Agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Tareas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Recursos/lista-de-verificacion.png</normaloff>:/Recursos/lista-de-verificacion.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223,224,229);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Nombre:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="inNombre">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                 </font>
                </property>
                <property name="accessibleName">
                 <string/>
                </property>
                <property name="inputMask">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>80</number>
                </property>
                <property name="placeholderText">
                 <string>Ingrese la tarea</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Fecha:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDateEdit" name="dateEdit">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="font">
            <font>
             <family>MS Reference Sans Serif</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="title">
            <string>Prioridad</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="inAlta">
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Alta</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="inMedia">
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Media</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="inBaja">
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Baja</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Tipo:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="inTipo">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cmdAgregar">
             <property name="font">
              <font>
               <family>MS Reference Sans Serif</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(224,224,224);</string>
             </property>
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="outDetalle_tareas">
           <property name="font">
            <font>
             <family>MS Reference Sans Serif</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(250, 251, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
